guide/getting-started/hardware.md

```diff
@@ -37,11 +37,12 @@ Illustration sources
 
 # Hardware overview
 
-Generally, any hardware with an internet connection can interact with the bitcoin network. While our personal computers and smartphones work great for general wallet management and transactions, a few types of dedicated hardware have been created that are optimized for the more unique aspects of how bitcoin works.
+Generally, any hardware that is connected to the Internet can interact with the Bitcoin network. While personal computers and smartphones work great for general wallet management and transactions, some dedicated hardware products are optimized for the nuances of Bitcoin. We’ll explore them and how they work here.
+
 
 ## Recovery phrase storage
 
-Simple, cheap, and robust storage solutions for recovery phrases come in subtle forms. [Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, for example, is a metal plate in which users can etch the words of their recovery phrase for long-term offline storage. [CardWallet](https://www.cardwallet.com){:target="_blank"}, in comparison, is the size of a credit card and comes pre-loaded with a recovery phrase (hidden behind a security seal) and a matching bitcoin address. And if you like an even simpler solution, a piece of paper will do. Although paper is not as durable and may be easy to lose, it is one of the strengths of Bitcoin that such simple solutions are possible.
+Simple, cheap, and robust storage solutions for recovery phrases come in many forms. [Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, is a metal plate that users can etch their recovery phrase into for long-term offline storage. [CardWallet](https://www.cardwallet.com){:target="_blank"}, by comparison, is the size of a credit card and pre-loaded with a recovery phrase (hidden behind a security seal) and a matching Bitcoin address. For an even simpler solution, a piece of paper will do. Although paper is less durable and easy to lose, it is a strength of Bitcoin that such simple solutions are possible.
```

Comment:
Adding comment by @sqcken for future reference or to discuss and improve right away in a review process.

> Feels like we should be saying more and covering more recovery storage solutions? No mention of Cobo type steel wallets?

Reply:
```suggestion
Simple, cheap, and robust storage solutions for recovery phrases come in many forms. [Seedplate](https://bitcoinseedbackup.com/){:target="_blank"}, is a metal plate that users can etch their recovery phrase into for long-term offline storage. For an even simpler solution, a piece of paper will do. Although paper is less durable and easy to lose, it is a strength of Bitcoin that such simple solutions are possible.
```
I would remove the point on CardWallet, just read their website and the private keys are generated by the facility, not by the user, major security vulnerability. https://www.cardwallet.com/en/faq/ 

If you wanted to add another point, just for fun and not recommended of course, you could say the human brain is also a hardware recovery phrase storage option 😆

Reply:
cc @GBKS

Reply:
For Ken's comment, I don't see a huge difference between Seedplate and the Cobo steel wallet. In one, you etch the words, on the other you put together small metal pieces with characters inscribed, but either way it's a steel backup.

Good point about removing the Cardwallet.

How about we link this to the ["Single metal backup" section of the "Bitcoin backups" page](https://bitcoin.design/guide/private-key-management/backups/#single-metal-backup), which is a hands-on guide for specifically these products. We could then add more links to reference products on that page, which would be appropriate for a hands-on guide, IMHO.

Reply:
Good catch.

Do you have a concrete suggestion here? Do we just re-write this section to mention paper and metal that link back to the bitcoin backups page which indeed provides good info

Reply:
I reckon we could just say something a long the lines of "various metal plate seed storage products exists and they are good for xyz reason." No need to mention seedplate / cobo plate - they are all essentially the same.

Reply:
As this change/conversation was outdated, I created a new change suggestion with the link to the backups page.